framework/core/src/Foundation/Container.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Foundation;
+
+class Container extends \Illuminate\Container\Container
+{
+    public function terminating(): void
+    {
+    }
+
+    public function terminate(): void
+    {
+    }
```

Comment:
I don't see these methods in the Illuminate Container, what are they for?

Reply:
The illuminate container in laravel is the same as the laravel application, so some internal service providers in laravel call app->terminating().

The only way to deal with that is to either re-declare said providers (too much boilerplate) or declare these placeholder methods to that it won't error out.

This is also part of why I believe we should re-couple back into a more laravel like architecture.